Editor/ProjectAuditorSettings.cs

```diff
@@ -27,5 +27,10 @@ public class ProjectAuditorSettings : ScriptableObject
         /// Maximum size of all files in the StreamingAssets folder. Beyond that size we report an issue.
         /// </summary>
         public int StreamingAssetsFolderSizeLimit = 50;
+
+        /// <summary>
+        /// Maximum size when it is not necessary to enable Streaming Mipmaps. Beyond that size we report an issue.
+        /// </summary>
+        public int TextureStreamingMipmapsMinimumSize = 1000;
```

Comment:
Just something Marco also pointed out on my last PR:

"TextureStreamingMipmapsSizeLimit" would be a better name to match the others, so then all "maximum size" values in this class have the same naming convention.